projects/css-custom-vars-viewer/src/CssTable.ts

```diff
@@ -149,6 +145,7 @@ export class CssTable extends SpectrumElement {
             <sp-table
                 size="m"
                 scroller="true"
+                style="height: 422px"
```

Comment:
Static values here seem like a bad idea. Can this get applied in CSS, or even better a Custom Property?

Reply:
I'm not sure yet, since I'm waiting for the sp table update to drop ^^; I wasn't planning on keeping it this way forever, but I do think that the style may have to be inline. 

To your first question, what files are missing? I just wanted to add the project; adding to the docs site will be another PR.

Reply:
It may just be a question of how the branches are made, but did you want this PR to open all of your additions to review (so the whole `projects/css-custom-vars-viewer` directory), or just the 5 files that are currently available?

Reply:
Also, you shouldn't need to wait for the Table updates anymore as being _in_ the repo means you get the latest changes even before they are released.

Reply:
Wow. I have no idea why the rest of the files didn't get added. Yikes. Hm.

Reply:
They're in the base branch, so it's likely you just created new branches at slightly less opportune times that you could have? 🤔